Example/PrebidDemoJava/src/main/java/org/prebid/mobile/app/MainActivity.java

```diff
@@ -269,6 +269,10 @@ public void showAd(View view) {
                     int refreshMillis = Integer.valueOf(refreshMillisString);
                     intent.putExtra(Constants.AUTO_REFRESH_NAME, refreshMillis);
                 }
+            } else if (adType.equals(getString(R.string.adTypeInAppNative)) && adServer.equals(getString(R.string.adServerAdManager))) {
```

Comment:
Add another Activity that uses a PublisherAdView to load in app native? Not sure how to put that in the UI yet.

Reply:
Or we can skip it in the demo, but make sure the docs contain this info.

Reply:
Okay, will add that to the doc.